src/main/java/com/sinthoras/visualprospecting/mixinplugin/Mixin.java

```diff
@@ -35,6 +35,9 @@
 
     GuiMapMixin("xaerosworldmap.GuiMapMixin", Side.CLIENT, XAEROWORLDMAP),
     WaypointsIngameRendererMixin("xaerosminimap.WaypointsIngameRendererMixin", Side.CLIENT, XAEROMINIMAP),
+    MinimapRendererMixin("xaerosminimap.MinimapRendererMixin", Side.CLIENT, XAEROMINIMAP, XAEROWORLDMAP),
```

Comment:
Are those really dependant on both mods? I did untangle your dependencies such that a player can install either or and it still works.

Reply:
Probably not? But do consider that it is currently not possible to make any overlays display without a world map mod